tracee-ebpf/tracee/tracee.bpf.c

```diff
@@ -2588,7 +2619,6 @@ int tracepoint__cgroup__cgroup_attach_task(struct bpf_raw_tracepoint_args *ctx)
     save_str_to_buf(&data, comm, 1);
     save_to_submit_buf(&data, (void*)&pid, sizeof(int), 2);
     events_perf_submit(&data, CGROUP_ATTACH_TASK, 0);
-
```

Comment:
No need to remove this line

Reply:
ok i just rebased so i think it solved that

Reply:
still removed